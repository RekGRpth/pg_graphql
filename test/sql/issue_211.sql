begin;

    create table project(
        id serial primary key,
        name varchar(255) not null
    );

    select graphql.resolve($$
    mutation {
      insertIntoProjectCollection(objects: [
        { name: "aaaa%aaaa"},
      ]) {
        affectedCount
        records {
          id
          name
        }
      }
    }
    $$);

rollback;
