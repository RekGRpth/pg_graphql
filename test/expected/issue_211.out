begin;
    create table project(
        id serial primary key,
        name varchar(255) not null
    );
    select graphql.resolve($$
    mutation {
      insertIntoProjectCollection(objects: [
        { name: "aaaa%aaaa"},
      ]) {
        affectedCount
        records {
          id
          name
        }
      }
    }
    $$);
                                                   resolve                                                    
--------------------------------------------------------------------------------------------------------------
 {"data": {"insertIntoProjectCollection": {"records": [{"id": 1, "name": "aaaa%aaaa"}], "affectedCount": 1}}}
(1 row)

rollback;
